report/renderer/category-renderer.js

```diff
@@ -92,6 +92,7 @@ export class CategoryRenderer {
         const snippets = this.dom.convertMarkdownLinkSnippets(pack.description);
         const packElm = this.dom.createElement('div', 'lh-audit__stackpack');
         packElm.append(packElmImg, snippets);
+        packElm.title = `Recommendation specific to ${pack.title} detected on the page`;
```

Comment:
Yeah I was looking at it just now and I agree; we've one more string at L82 that isn't i18ned.

Reply:
Ah, I think the reason that isn't formatted is because the strings can change based on runtime values. We would need to call `formatString` here to translate this.

The flow report already does this in a few places but the normal report renderer does not:
https://github.com/GoogleChrome/lighthouse/blob/main/flow-report/src/i18n/i18n.tsx#L50